test/domain_test.py

```diff
@@ -21,7 +21,53 @@
 )
 
 
-class BuildJujuPodSpecTest(unittest.TestCase):
+class PrometheusServiceConfigAndPodSpecTest(unittest.TestCase):
+    def setUp(self):
+        self.default_mock_config = {
```

Comment:
these should probably be named `default_cli_args`

Reply:
No, because it's the whole charm config, not the CLI args only (however, at the moment CLI args are 90% of this config, but that might change soon :)